JetUtilities/interface/crystalBall.h

```diff
@@ -0,0 +1,7 @@
+#ifndef CRYSTALBALL_HH
+#define CRYSTALBALL_HH
+
+double crystalBall(double *xx, double *pp); // global fit using the function above
+double crystalBall_1(double *xx, double *pp); // global fit using the function above
```

Comment:
crystalBall uses a polynome of order 7 (for tauES) and crystalBall_1 a polynome of order 1 (for tau transfer function).